### ISSUE TITLE
External CTs not showing when no generator connected

### ISSUE THREAD

**Issue by skipfire** [jgyates/genmon#668]
I'm working on some improvements to the CTs, but the data is not showing with the current version.

### Expected Behavior

When the External CT add-on is enabled and Strict is OFF, the gauges and CT data would show with or without a generator present.

### Actual Behavior

Not seeing the External CT gauges or data in a fresh genmon install that has never been connected to a generator even with the add-on enabled, strict off, and the Sensor HAT connected.

### Steps to Reproduce (including precondition)

Enable the add-on, turn strict off, wait for genmon to restart, view page.  This happened with a fresh install as well as an install that I've previously been using where the CT data was visible.

### Screenshot or Pictures relating to the problem (if possible)

![image](https://user-images.githubusercontent.com/10307327/152650149-e0aae555-1504-4b8c-a36f-577fec751275.png)
![image](https://user-images.githubusercontent.com/10307327/152650157-206d8eb0-4a7e-4204-84e5-e42c71ba86eb.png)

### Your Environment

- Generator Model: none connected
- Genmon Logs: submitted from 2 different installs
- Genmon Version: v1.18.05 (installed around 2022-02-05 11:00 AM EST)

**Comment by jgyates** [comment]
I can not duplicate this. Also, I have not received any logs on this, do you have outbound email enabled? Looking at /var/log/gencthat.log may provide some insights. Have you edited /boot/config.txt?

**Comment by jgyates** [comment]
I can duplicate it now. i will let you now what I find.

**Comment by skipfire** [comment]
I had edited /boot/config.txt and tested with my stand alone script so I know that it was reading from the HAT.  After that I installed GenMon and my setting is gone and the line  where it was (immediately above the [CM4] which was the first type tag) is where I think genmon install added the top dtoverlay=pi3-disable-bt.  After adding the spi1-1cs line back in I got the text for the legs but not the gauges for the CTs, CPU, or WiFi, but I'm not worried about the gauges since I can see the legs.
